pkg/cli/mirror/mirror.go

```diff
@@ -445,10 +445,9 @@ func (o *MirrorOptions) mirrorMappings(cfg v1alpha2.ImageSetConfiguration, image
 			Type: srcRef.Type,
 		}
 
-		dstTIR := imagesource.TypedImageReference{
-			Ref:  dstRef.Ref,
-			Type: dstRef.Type,
-		}
+		// OCPBUGS-11922
+		dstTIR := o.processNestedPaths(&dstRef)
+
```

Comment:
Although this works, and mirrors correctly the images, the original TypedImageMapping (`images`) is not modified, and is not returned by this method. 
Therefore, the mapping.txt that is written L818 doesn't contain the renamed images, and nore will the ImageContentSourcePolicy that is generated from it.